sirepo/package_data/static/js/raydata.js

```diff
@@ -624,6 +630,20 @@ SIREPO.app.directive('scansTable', function() {
                 ];
             };
 
+            $scope.setDefaultStartStopTime = () => {
+                const m = appState.models[$scope.modelName];
+                if (!m.searchStartTime && !m.searchStopTime) {
+                    $scope.setSearchTimeLastHour();
+                };
+            };
+
+            $scope.setSearchTimeLastHour = () => {
```

Comment:
I separated this out because I anticipate using it for https://github.com/radiasoft/sirepo/issues/6059. It would probably be a good idea to get this merged quickly so I can start that issue with this change